test/WebJobs.Script.Tests.Integration/Storage/AzureTableStorageProviderTests.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Azure.Data.Tables;
+using Microsoft.Extensions.Configuration;
+using Xunit;
+
+namespace Microsoft.Azure.WebJobs.Script.Tests.Integration.Storage
+{
+    public class AzureTableStorageProviderTests
+    {
+        private const string StorageConnection = "AzureWebJobsStorage";
+
+        [Fact]
+        public async Task TestAzureTableStorageProvider_ConnectionInWebHostConfiguration()
```

Comment:
nit: the usual pattern is for the test method name to start with the method being tested, e.g. here it'd be TryCreateHostingTableServiceClient_ConnectionInWebHostConfiguration.

Same comment for the other test methods below